modules/ztp-pgt-config-best-practices.adoc

```diff
@@ -15,3 +15,10 @@ Consider the following best practices when customizing site configuration `{poli
 * `MachineConfig` CRs should be included as `extraManifests` in the `SiteConfig` CR so that they are applied during installation. This can reduce the overall time taken until the cluster is ready to deploy applications.
 
 * `{policy-gen-cr}` CRs should override the channel field to explicitly identify the desired version. This ensures that changes in the source CR during upgrades does not update the generated subscription.
+
+* The default setting for `policyDefaults.consolidateManifests` is `true`. This is the recommended setting for DU profile. Setting it to `false` could impact large scale deployments.
```

Comment:
```suggestion
* The default setting for `policyDefaults.consolidateManifests` is `true`. This is the recommended setting for DU profile. Setting it to `false` might impact large scale deployments.
```
ISG word usage guidance for "could": https://www.ibm.com/docs/en/ibm-style?topic=word-usage#could